source/components/RadialList/styles.js

```diff
@@ -3,6 +3,7 @@ export default {
     display: 'block',
     position: 'relative',
     margin: 0,
-    padding: 0
+    padding: 0,
+    transition: 'all 500ms ease-in-out'
```

Comment:
Any reason for the transition all? If you need a couple more properties you can chain them easily.